habitat/core/embodied_task.py

```diff
@@ -389,3 +389,6 @@ def is_episode_active(self):
 
     def seed(self, seed: int) -> None:
         return
+
+    def get_metrics_at_episode_end(self):
```

Comment:
I find this a little bit awkward. I do not think this method fits into the embodied task class. Would it be possible to have a measure class instead? This way there would be no need to modify the get_metrics code to get the gfx. You could even directly write to the file inside of the measure (at each episode end).